idp_data/idp_data/migrations/0025_municipality_ward_councillor_url.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.10 on 2020-10-01 13:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('idp_data', '0024_municipality_by_laws_url'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='municipality',
+            name='ward_councillor_url',
+            field=models.CharField(default='', max_length=500, verbose_name='Find Your Ward Councillor Url'),
```

Comment:
Same here, `URL` not `Url` when displayed to users.

Reply:
created a new migration with `URL` keyword instead of `Url` or `url`